A/alsa/build_tarballs.jl

```diff
@@ -3,19 +3,18 @@
 using BinaryBuilder
 
 name = "alsa"
-# It's actually v1.2.5.1
-version = v"1.2.5"
+version = v"1.2.11"
 
 # Collection of sources required to complete build
 sources = [
-    ArchiveSource("https://www.alsa-project.org/files/pub/lib/alsa-lib-1.2.5.1.tar.bz2",
-                  "628421d950cecaf234de3f899d520c0a6923313c964ad751ffac081df331438e"),
+    ArchiveSource("https://www.alsa-project.org/files/pub/lib/alsa-lib-1.2.11.tar.bz2",
+                  "9f3f2f69b995f9ad37359072fbc69a3a88bfba081fc83e9be30e14662795bb4d"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 cd $WORKSPACE/srcdir/alsa-lib*/
-./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
+./configure --prefix=${prefix} --with-configdir=/usr/share/alsa --with-plugindir=/usr/lib/alsa-lib --build=${MACHTYPE} --host=${target}
```

Comment:
Would that work on nix?

Reply:
Probably not on nix, and this will also reference the user's existing Alsa stuff, not anything we provide in the JLL. There are apparently environment variables that could be used to work around this at runtime (they basically override this setting apparently).

Reply:
I was moderately sure we were doing this already

Reply:
#1432